### PR TITLE
feat(tier4_perception_launch): add image segmentation based pointcloud filter

### PR DESCRIPTION
**Pull request by badai-nguyen**
## Description
- This is to add image semantic segmentation based pointcloud filter into perception pipeline to sold the issue https://github.com/autowarefoundation/autoware.universe/issues/5594
- The default mode is still disable. Make sure the tensorrt_yolox output semantic segmentation mask image is output by merging https://github.com/autowarefoundation/autoware.universe/pull/5553 and https://github.com/autowarefoundation/autoware/pull/4012 when the filter is enable.

image

## Releated Link

- [TIER IV INTERNAL LINK](https://tier4.atlassian.net/browse/RT1-3590)

## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by technolojin — APPROVED** [review]
LGTM

**Review by YoshiRi — APPROVED** [review]
LGTM

File: launch/tier4_perception_launch/package.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@
   <maintainer email="yukihiro.saito@tier4.jp">Yukihiro Saito</maintainer>
   <maintainer email="shunsuke.miura@tier4.jp">Shunsuke Miura</maintainer>
   <maintainer email="yoshi.ri@tier4.jp">Yoshi Ri</maintainer>
+  <maintainer email="taekjin.lee@tier4.jp">Taekjin Lee</maintainer>
```
Comment (yukkysaito) [inline launch/tier4_perception_launch/package.xml:10]: I think it would be better to make these changes in a separate PR.

Comment (badai-nguyen) [reply launch/tier4_perception_launch/package.xml]: @yukkysaito I reverted that [commit ](https://github.com/autowarefoundation/autoware.universe/pull/7225/commits/31348794e250401d36006a1bcf250bda17ba8242)

**Review by yukkysaito — APPROVED** [review]
LGTM
